src/aiida/cmdline/groups/dynamic.py

```diff
@@ -88,29 +88,25 @@ def get_command(self, ctx: click.Context, cmd_name: str) -> click.Command | None
             command = super().get_command(ctx, cmd_name)
         return command
 
-    def call_command(self, ctx, cls, **kwargs):
+    def call_command(self, ctx, cls, non_interactive, **kwargs):
```

Comment:
Can you briefly explain `non_interactive`?

Reply:
This is a generic option that is used in all interactive `verdi` commands. It is the `-n/--non-interactive` flag, and when specified it forces the command to never prompt, which is necessary for use in non-interactive scripts for example.